### PR TITLE
Added SPonGE CasE

### PR DESCRIPTION
**Pull request by pernielsentikaer**
## Description

Added SPonGE CasE

## Screencast

CleanShot 2022-10-05 at 18 27 29@2x

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
CC @es183923 😊

File: extensions/change-case/package.json [hunk 1]
```diff
@@ -212,6 +215,15 @@
       "label": "Upper First",
       "default": true,
       "required": false
+    },
+    {
+      "name": "SPonGECasE",
```
Comment (es183923) [inline extensions/change-case/package.json]: should be `SpongeCase` to mantain consistency

File: extensions/change-case/package.json [hunk 2]
```diff
@@ -212,6 +215,15 @@
...
+      "type": "checkbox",
+      "title": "",
+      "description": "Enable the \"SPonGE CasE\" Transformation",
+      "label": "SponGE CaSE",
```
Comment (es183923) [inline extensions/change-case/package.json:224]: should be `Sponge Case` to mantain consistency

File: extensions/change-case/src/change-case.tsx [hunk 3]
```diff
@@ -35,7 +35,8 @@ type CaseType =
   | "Swap Case"
   | "Title Case"
   | "Upper Case"
-  | "Upper First";
+  | "Upper First"
+  | "SPonGE CasE";
```
Comment (es183923) [inline extensions/change-case/src/change-case.tsx]: should be `Sponge Case` to mantain consistency

File: extensions/change-case/src/change-case.tsx [hunk 4]
```diff
@@ -98,6 +99,7 @@ export default function changeChase() {
     { type: "Title Case", func: changeCase.titleCase },
     { type: "Upper Case", func: changeCase.upperCase },
     { type: "Upper First", func: changeCase.upperCaseFirst },
+    { type: "SPonGE CasE", func: changeCase.spongeCase },
```
Comment (es183923) [inline extensions/change-case/src/change-case.tsx]: same here

**Comment by es183923** [comment]
the only thing that should represent the casing should be the description

**Comment by pernielsentikaer** [comment]
Should all be better now @es183923 🔥

**Comment by es183923** [comment]
all good now

**Pull request merged**